test/epi_locator_web/controllers/admin_auth_test.exs

```diff
@@ -140,23 +140,23 @@ defmodule EpiLocatorWeb.AdminAuthTest do
 
     test "stores the path to redirect to on GET", %{conn: conn} do
       halted_conn =
-        %{conn | request_path: "/foo", query_string: ""}
+        %{conn | path_info: ["foo"], query_string: ""}
```

Comment:
I am not very comfortable with this change. It appears to rely on the internal structure of `conn` rather than using any public APIs. These tests failed precisely because the `conn` structure changed (I think, anyway). So I'd much rather find a way to set up the `conn` using public API, that would ideally pass on both versions of Phoenix.

The implementation uses public API so it didn't need to change - only the test setup changed.

Reply:
replaced with public API